packages/starlight/utils/navigation.ts

```diff
@@ -184,13 +184,17 @@ function getOrder(routeOrDir: Route | Dir): number {
 }
 
 /** Sort a directory’s entries by user-specified order or alphabetically if no order specified. */
-function sortDirEntries(dir: [string, Dir | Route][]): [string, Dir | Route][] {
-	return dir.sort(([, a], [, b]) => {
+function sortDirEntries(
+	dir: [string, Dir | Route][],
+	locale: string | undefined
+): [string, Dir | Route][] {
+	const collator = new Intl.Collator(localeToLang(locale));
```

Comment:
Really nice! I didn’t know about `Intl.Collator` — so nice how much i18n stuff like this is built into Node these days.

Reply:
Definitely, and super handy. I think that is what is used under the hood by the String [`localeCompare()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare) but they recommend creating an `Intl.Collator` object for repeated use.

Just to double-check for sanity, I can safely merge this one as it is a patch and it does not update the docs, right?

Reply:
> I can safely merge this one as it is a patch and it does not update the docs, right?

Yes! Safe to merge.